src/ontology/edits/oeo-social.omn

```diff
@@ -2399,6 +2428,8 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/944
 move govern axioms to oeo-shared-axioms
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1399
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1724",
+        rdfs:comment "EU-EHS"@de,
+        rdfs:comment "Emissionshandelssystem der Europäischen Union"@de,
```

Comment:
These two should be implemented as `alternative label` and not as `rdfs:comment`